app/assets/javascripts/abstract-list.js

```diff
@@ -288,28 +291,25 @@ require(["lib/models", "lib/tools", "knockout", "sammy", "lib/offline"], functio
         };
 
         self.isFavourite = function(abstract) {
-            var uuid = abstract.uuid;
-            var favUsersUrl = "/api/user/self/abstract/" + uuid + "/isfavuser";
```

Comment:
If this is the only occasion where the `isfavuser` route is used, remove it from the backend as well.